app/controllers/twilio_controller.rb

```diff
@@ -55,34 +47,43 @@ def connect
     # format. Our Ruby library provides a helper for generating one
     # of these documents
     response = Twilio::TwiML::Response.new do |r|
-      r.Say 'Thanks for contacting our sales department. Our ' +
-        'next available representative will take your call.', :voice => 'alice'
+      r.Say 'Thanks for contacting our sales department. Our '\
```

Comment:
Shouldn't we use parentheses when we have more than a single argument to a function call?